docs/software/integrations/mqtt/mosquitto.mdx

```diff
@@ -47,4 +47,54 @@ sidebar_position: 1
    brew services restart mosquitto
    ```
 
-7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
\ No newline at end of file
+7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
+
+
+### Using mosquitto on raspberry pi bookworm
+
+1. install mqtt server
+
+   ```sh
+   sudo apt-get install mosquitto mosquitto-clients
+   ```
+
+2. start the mqtt server
+
+   ```sh
+   sudo service mosquitto start
+   ```
+
+3. Do a quick test of server, start a subscriber on a topic:
+
+   Note: this will wait until you press control-c (publish a message, see below)
+
+   ```sh
+   mosquitto_sub -t test/hello
+   ```
+
+4. In another console, publish a message to that topic:
+
+   ```sh
+   mosquitto_pub -h localhost -q 0 -t test/hello -m 'yo!'
+   ```
+
+5. For Meshtastic to be able to access that server, two settings need to be changed in the
+   `/etc/mosquitto/conf.d/mosquitto.conf` file:
+
+   ```shell
+   sudo sh -c "echo 'listener 1883 0.0.0.0\nallow_anonymous true' >> /etc/mosquitto/conf.d/mosquitto.conf"
+   ```
+
+6. Restart the service:
+
+   ```shell
+   sudo service mosquitto restart
+   ```
+
+Optionally pull a copy of [pdxlocations/MQTT-Connect-for-Meshtastic](https://github.com/pdxlocations/MQTT-Connect-for-Meshtastic)
```

Comment:
I'm blushing but not sure if this is the place to link a community project?

Reply:
its a very specific client for this purpose and getting it going includes a lot of how-to - your call should I huck it?